src/languages/koCSSLanguage.py

```diff
@@ -52,6 +53,14 @@ class koCSSCommonLanguage(KoLanguageBase):
     searchURL = "http://www.google.com/search?q=site%3Ahttp%3A%2F%2Fwww.w3schools.com%2Fcss+%W"
 
     sample = """
+
```

Comment:
I dont think this should be part of your PR? Same for the bit below.

For one thing /fonts wont exist. And these modifications seem pointless.